cmd/util/ledger/migrations/storage_v4.go

```diff
@@ -223,6 +225,75 @@ func rencodeValueV4(data []byte, owner common.Address, key string, version uint1
 	return newData, nil
 }
 
+var flowTokenReceiverStorageKey = interpreter.StorageKey(
+	interpreter.PathValue{
+		Domain:     common.PathDomainStorage,
+		Identifier: "flowTokenReceiver",
+	},
+)
+
+var flowTokenForwardingLocation = common.AddressLocation{
+	Address: common.BytesToAddress([]byte{0x75, 0x4a, 0xed, 0x9d, 0xe6, 0x19, 0x76, 0x41}),
```

Comment:
this is probably different on mainnet and testnet?

Reply:
Yes, but I don't know the value for Testnet and Mainnet yet, I'll look for it

Reply:
added support for Mainnet in 5148df6